test/e2e/storage/utils/utils.go

```diff
@@ -182,6 +190,10 @@ func TestVolumeUnmountsFromDeletedPodWithForceOption(c clientset.Interface, f *f
 		err = c.CoreV1().Pods(clientPod.Namespace).Delete(clientPod.Name, &metav1.DeleteOptions{})
 	}
 	Expect(err).NotTo(HaveOccurred())
+
+	// Wait for pod to enter "Terminating state"
+	time.Sleep(30 * time.Second)
```

Comment:
Can we use one of the fancy API Machinery wait functions here to wait for that specific condition. Like a wait.Until (not sure if thats the exact call)

Reply:
Let me see if this is actually needed in the cherry pick, as this call was added in master originally but not backported here.